src/xFactor/Participant.java

```diff
@@ -0,0 +1,57 @@
+package xFactor;
+
+import java.util.ArrayList;
+
+public class Participant extends Person {
+	private static int countParticipants;
+	private String quality;

```

Comment:
make participant have multiple qualities. They can be predefined list, or even new objects